rasterio/_env.pyx

```diff
@@ -57,6 +59,8 @@ code_map = {
 
 log = logging.getLogger(__name__)
 
+os.environ.setdefault("CURL_CA_BUNDLE", certifi.where())
+os.environ.setdefault("PROJ_CURL_CA_BUNDLE", certifi.where())
```

Comment:
This will have no effect on the tests in rasterio-wheels.

Reply:
@snowman2 I suppose rasterio and fiona could call the PROJ API function like pyproj does, but for now I'm trying to only use the GDAL API.

Tests with OS X at https://travis-ci.com/github/rasterio/rasterio-wheels/builds/188441746 all pass. I think this could work.

Reply:
It might be worthwhile to add methods to GDAL for setting the CA Bundle Path for both GDAL and PROJ.